Makefile

```diff
@@ -1,13 +1,15 @@
-STACK := "scalingo-20"
-IMAGE := scalingo/$(STACK):latest
-BASH_COMMAND := /bin/bash
+STACK = "scalingo-20"
```

Comment:
question: what is the difference between `=` and `:=`?

Reply:
Ah! 
- `:=` is the assignment operator for what they call "Simply Expanded" variables.
  It means that `make` will evaluate these variables only once.
- `=` is the assignment operator for they call "Recursively Expanded" variables.
  Contrary to the "Simply Expanded" variables, these ones will be evaluated each time they are used.

Example, with this `Makefile`:
```make
default: target1 target2

target1 target2:
    @echo "Simple:    $(SPL)"
    @echo "Recursive: $(REC)"
    @sleep 10s
    @echo "Simple:    $(SPL)"
    @echo "Recursive: $(REC)"
    @echo

SPL := $(shell date +%s.%N)    # Executed once, so value is constant
REC = $(shell date +%s.%N)     # Executed on each call, value will change
```

You'll get something like this as a result:
```shell
% make        
Simple:    1705675113.571677216
Recursive: 1705675113.576677823
Simple:    1705675113.571677216
Recursive: 1705675113.580262060

Simple:    1705675113.571677216
Recursive: 1705675123.601350834
Simple:    1705675113.571677216
Recursive: 1705675123.604350785
```

As you can see, the _Simple_ value never changes, while the _Recursive_ one is always re-evaluated.

I used this to do some tests, I guess we can switch back to `:=`.